### PR TITLE
#1970 - File integration - Fix the incorrect line number of detail records

### PR DESCRIPTION
**Pull request by andrepestana-aot**
- Changed the code to take into account the header file line number when counting the file content line numbers.
- Fixed broken tests

### CODE REVIEW

File: sources/packages/backend/libs/integrations/src/cra-integration/cra.integration.service.ts [hunk 1]
```diff
@@ -149,7 +149,7 @@ export class CRAIntegrationService extends SFTPIntegrationBase<CRASFTPResponseFi
     const statusRecords: CRAResponseStatusRecord[] = [];
     const totalIncomeRecords: CRAResponseTotalIncomeRecord[] = [];
     fileLines.forEach((line: string, index: number) => {
-      const lineNumber = index + 1;
+      const lineNumber = index + 2; // Take into account the removed header line.
```
Comment (ann-aot) [inline sources/packages/backend/libs/integrations/src/cra-integration/cra.integration.service.ts:152]: maybe we  can move  the comment above the line 152

**Review by ann-aot — APPROVED** [review]
LGTM 👍 Just a minor comment added

File: sources/packages/backend/libs/integrations/src/cra-integration/cra.integration.service.ts (hunk 1 above)
Comment (andrewsignori-aot) [reply sources/packages/backend/libs/integrations/src/cra-integration/cra.integration.service.ts:152]: It would be ok for me either way 😉

**Review by andrewsignori-aot — APPROVED** [review]
Looks good 👍

**Review by dheepak-aot — APPROVED** [review]
LGTM 👍

**Pull request merged**